app/lib/internet_archive/archival.rb

```diff
@@ -0,0 +1,48 @@
+# Uses the Save Page Now 2 API
+# (https://docs.google.com/document/d/1Nsv52MvSjbLb2PCpHlat0gkzw0EvtSgpKHu4mk0MnrA)
+class InternetArchive::Archival
+  attr_reader :website_url, :job_id
+
+  def initialize(website_url)
+    @website_url = website_url
+  end
+
+  def request
```

Comment:
It could be nice to call this method `capture` since it's more specific and is the term that [SPN2 uses](https://docs.google.com/document/d/1Nsv52MvSjbLb2PCpHlat0gkzw0EvtSgpKHu4mk0MnrA/edit#heading=h.uu61fictja6r)

Reply:
Then it's capturing an archival, which is kinda confusing. I think in this case we should call it an `InternetArchive::Capture`, which makes more sense anyway, and then we can `request` that.

Reply:
yeah i'm cool with that too